cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionProposalTest.java

```diff
@@ -4,35 +4,54 @@
 
 package com.linkedin.kafka.cruisecontrol.executor;
 
+import com.linkedin.kafka.cruisecontrol.model.ReplicaPlacementInfo;
```

Comment:
The subset of changes in this test can be moved to a patch that applies to `master`. Can we add them to the relevant patch?

Reply:
`master` branch does not have ReplicaPlacementInfo since there is no logdir information for `Replica`.

Reply:
Can we move the broker ids such as `1` to variables as we did here? I.e. let's do the part#2 of: (1) using `ReplicaPlacementInfo`, and (2) moving the hardcoded values to fields in `master`.